### PR TITLE
WTF blog post draft

### PR DESCRIPTION
**Pull request by karawoo**
(this PR is intentionally just to another branch within my fork, to give @jennybc and @jimhester a place to put feedback before we submit the post)

### CODE REVIEW

File: content/blog/2020-02-09-rstudio-conf-2020-what-they-forgot/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+[debugging](https://rstudio-conf-2020.github.io/what-they-forgot/day1_3)
+tools, including `traceback()`, `browser()`, `debug()`, `trace()`, and
+`recover()`. Finally, we covered how to use
+[git and GitHub](https://rstudio-conf-2020.github.io/what-they-forgot/day1_4)
+to track the history of our projects and communicate our results.
+
+Day two began with [personal R administration](https://rstudio-conf-2020.github.io/what-they-forgot/day2_1):
+how to maintain your R installation and package libraries, how to set up a
+development environment, how to install source packages and read source code,
+and how to create reproducible project environments with renv. From there, Jenny
```
Comment (jimhester) [inline content/blog/2020-02-09-rstudio-conf-2020-what-they-forgot/index.md:42]: Do we want a link to https://rstudio-conf-2020.github.io/what-they-forgot/day2_2 anywhere in here?

**Review by jimhester — APPROVED** [review]
One minor question, otherwise I think it is great, thanks for taking the time to write it up @karawoo!

File: content/blog/2020-02-09-rstudio-conf-2020-what-they-forgot/index.md (hunk 1 above)
Comment (karawoo) [reply content/blog/2020-02-09-rstudio-conf-2020-what-they-forgot/index.md:42]: Yeah I was thinking about that but wasn't sure where to put the link. Maybe I should make it "[personal R administration](https://rstudio-conf-2020.github.io/what-they-forgot/day2_1) ([part 2](https://rstudio-conf-2020.github.io/what-they-forgot/day2_2))"?

**Comment by jennybc** [comment]
As conveyed elsewhere, LGTM. Thanks!

**Review by jennybc — APPROVED** [review]

File: content/blog/2020-02-09-rstudio-conf-2020-what-they-forgot/index.md (hunk 1 above)
Comment (jimhester) [reply content/blog/2020-02-09-rstudio-conf-2020-what-they-forgot/index.md:42]: yeah that sounds good

**Pull request merged**